### PR TITLE
fix: Couple of tests are permanently broken while running locally

### LINKED ISSUE #472: Couple of tests are permanently broken while running locally
Fail logs:
```
______________________TypesTests.test_TimeFromTicks______________________ 

self = <tests.spanner_dbapi.test_types.TypesTests testMethod=test_TimeFromTicks>

    def test_TimeFromTicks(self):
        epochTicks = 1572851662.9782631  # Sun Nov 03 23:14:22 2019
        got = TimeFromTicks(epochTicks)
        # Since continuous integration infrastructure such as Travis CI
        # uses clocks on UTC, it is useful to be able to compare against
        # either of UTC or the known standard time.
        want = (
            datetime.time(23, 14, 22),
            datetime.datetime(2019, 11, 4, 7, 14, 22, tzUTC).time(),
        )
        matches = got in want
>       self.assertTrue(matches, '`%s` not present in any of\n`%s`' % (got, want))
E       AssertionError: False is not true : `10:14:22` not present in any of
E       `(datetime.time(23, 14, 22), datetime.time(7, 14, 22))`

tests\spanner_dbapi\test_types.py:58: AssertionError




______________________TypesTests.test_TimestampFromTicks______________________

self = <tests.spanner_dbapi.test_types.TypesTests testMethod=test_TimestampFromTicks>

    def test_TimestampFromTicks(self):
        epochTicks = 1572851662.9782631  # Sun Nov 03 23:14:22 2019
        got = TimestampFromTicks(epochTicks)
        # Since continuous integration infrastructure such as Travis CI
        # uses clocks on UTC, it is useful to be able to compare against
        # either of UTC or the known standard time.
        want = (
            datetime.datetime(2019, 11, 3, 23, 14, 22),
            datetime.datetime(2019, 11, 4, 7, 14, 22, tzUTC),
        )
        matches = got in want
>       self.assertTrue(matches, '`%s` not present in any of\n`%s`' % (got, want))
E       AssertionError: False is not true : `2019-11-04 10:14:22` not present in any of
E       `(datetime.datetime(2019, 11, 3, 23, 14, 22), datetime.datetime(2019, 11, 4, 7, 14, 22))`

tests\spanner_dbapi\test_types.py:71: AssertionError
```

### PR DESCRIPTION
**Pull request by tina80lvl**
Fixes: https://github.com/googleapis/python-spanner-django/issues/472

### CODE REVIEW

**Comment by tina80lvl** [comment]
Previous value 1572851662.9782631 corresponded 04.11.2019 10:14:22, not 03.11.2019 23:14:22 as written in comments

**Review by mf2199 — COMMENTED** [review]
The epoch timestamp [seconds] for `Sunday, 3 November 2019 23:14:22 UTC` is `1572822862`, not `1572812062`, as can be verified [here](https://www.epochconverter.com/?prefs). So either the value or the comment needs to be updated.

**Comment by mf2199** [comment]
@tina80lvl Could you please also add a meaningful description to this PR. If it refers to an issue, then somewhere in that description should be a reference to it in a form of `Fixes: #[issue-number]`. The description can later be transfered to the corresponding public PR, so there is no danger of duplicating the effort.

**Comment by tina80lvl** [comment]
> The epoch timestamp [seconds] for `Sunday, 3 November 2019 23:14:22 UTC` is `1572822862`, not `1572812062`, as can be verified [here](https://www.epochconverter.com/?prefs). So either the value or the comment needs to be updated.

I'v found the problem. `1572812062` will be correct time in GMT+03:00, but absolute correct time is `1572822862`. Test covers both values with GTM and without but I guess here can be mistake, I'll fix

File: tests/spanner_dbapi/test_types.py [hunk 1]
```diff
@@ -17,8 +18,8 @@
 )
 from google.cloud.spanner_dbapi.utils import PeekIterator
 
-tzUTC = 0  # 0 hours offset from UTC
```
Comment (IlyaFaer) [inline tests/spanner_dbapi/test_types.py:20]: Deleting this whitespace seems to be wrong. Black sets comments with two whitespaces
It's a standard:
https://www.python.org/dev/peps/pep-0008/#inline-comments

**Review by IlyaFaer — APPROVED** [review]
Okay, as it's seems to be working fine, I think we can create a public PR

**Pull request merged**